dictionaries/ui/components/modal/en.dictionary.itop.modal.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright (C) 2013-2021 Combodo SARL
+ *
+ * This file is part of iTop.
+ *
+ * iTop is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * iTop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ */
+
+// Input
+Dict::Add('EN US', 'English', 'English', array(
+	'UI:Modal:DefaultConfirmationTitle' => 'Confirmation',
```

Comment:
```suggestion
	'UI:Modal:Confirmation:DefaultTitle' => 'Confirmation',
```

As we might have more and more predefined modals, what to think of namespacing things like this?
This way we can add more and more modals in the future and still find what is related to each easily.